test/unit-test.js

```diff
@@ -0,0 +1,46 @@
+var expect = require('chai').expect;
+var assert = require('chai').assert;
+var should = require('chai').should;
+
+const fs = require('fs');
+const h5 = require('../index')
+const chai = require('chai')
```

Comment:
I just went through these and we're using assert and expect, so I destructured like you suggested. However, I've got to keep that full `const chai = require('chai')` in there so that I can later call `chai.use(chaiAsPromised)`, unless you have another way you think I should do that?